test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/011_App_version_is_more_then_version_deprecated_params_in_DB_and_API.lua

```diff
@@ -0,0 +1,50 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: The app is NOT able to retrieve the parameter in case app version is less then parameter version,
+-- parameter is listed in DB and API
+-- In case:
+-- 1) App is registered with syncMsgVersion=6.0
+-- 2) `FuelRange` parameter has since=5.0 in API and DB
+-- 3) App requests GetVehicleData(fuelLevel and fuelLevel_State)
+-- SDL does:
+-- 1) reject the request as empty one
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/common')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 5
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+
+local fuelLevelParams = {
+  fuelLevel = 106 ,
+}
+
+local fuelLevelStateParams = {
+  fuelLevel_State = "NORMAL",
+}
+
+--[[ Local Functions ]]
+local function processGetVDunsuccess(pParams)
+  local cid = common.getMobileSession():SendRPC("GetVehicleData", pParams)
```

Comment:
GetVD request should contain the request as `parameterName=true`.
In case of deprecated parameter  mobile should be ABLE to retrieve the parameter in this case.